sources/web/datalab/jupyter.ts

```diff
@@ -304,8 +305,14 @@ function responseHandler(proxyResponse: http.ClientResponse,
 
       sendTemplate('tree', templateData, response);
       page = 'tree';
-    }
-    else {
+    } else if (path.indexOf('/edit') == 0) {
+      // stripping off the /edit/ from the path
+      templateData['filePath'] = path.substr(6);
+      templateData['fileName'] = path.substr(path.lastIndexOf('/') + 1);
+
+      sendTemplate('edit', templateData, response);
+      page = 'edit';
```

Comment:
You're right; it seems to have been a half-completed cleanup to remove the duplicated calls to `sendTemplate`.

Rather than delete the variable, I'm going to complete the cleanup and remove the duplicated calls to `sendTemplate`.